homeworks/nazar_kostiv/pyats01_hw.py

```diff
@@ -3,46 +3,34 @@
 from pyats.aetest import Testcase, test, main
 from calculation import add, divide
 
-parameters = {
-        'num1': 3,
-        'num2': 0
-    }
 
 class SmokeTest(Testcase):
-    parameters = {
-        'num1': 3,
-        'num2': 0
-    }
 
     @test
     def test_add(self, steps, num1, num2):
```

Comment:
"self" is not used, so is not required
```suggestion
    def test_add(steps, num1, num2):
```